examples/querying_entity_versions.ipynb

```diff
@@ -0,0 +1,782 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ec69ae04-1895-4243-a949-71360175b173",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "Now we have an entity reference for our logo, we can use the API to learn more about it.\n",
+    "\n",
+    "Versioning information is [not part of the core API](https://github.com/OpenAssetIO/OpenAssetIO/blob/main/doc/decisions/DR017-Entity-version-queries.md) itself. MediaCreation traits that describe how we generally work with versions in the post-production industry are used with the core API methods."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4348da46-9467-4734-9f96-d84e26f7c11f",
+   "metadata": {
+    "editable": true,
+    "jp-MarkdownHeadingCollapsed": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "## Querying an entity's version"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5e32a587-5233-441e-9b80-1b2d6d3769fb",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "An entity's version can be resolved using the imaginatively named `VersionTrait` from the `lifecycle` module of the `openassetio_mediacreation` package `traits`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "0fcf0214-2561-488a-8eee-0330ca27710d",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "from openassetio_mediacreation.traits.lifecycle import VersionTrait\n",
+    "from openassetio.access import ResolveAccess\n",
+    "\n",
+    "entity_data = manager.resolve(logo_ref, {VersionTrait.kId}, ResolveAccess.kRead, context)\n",
+    "\n",
+    "version_trait = VersionTrait(entity_data)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a6c4f0ff-0e5d-40b7-84d6-d46efa8cba7a",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "We use the `VersionTrait` class as a wrapper around the generic `TraitsData` returned from the API to inspect it's properties - which contain information about the entity's version. This helps ensure we don't accidentally mis-spell traits or their properties.\n",
+    "\n",
+    "MediaCreation use the term \"tag\" for some string that identifies the version of an entity. Using a string, rather than a number allows full flexibility in how a specific manager may wish to handle versions and dynamic \"meta-versions\".\n",
+    "\n",
+    "**Note: that the tags themselves are specific to each manager, and are not defined by the API or the MediaCreation library. Hosts should not attempt to string munge entity references, and use the methods covered in this document instead if they wish to obtain references for specific versions. Otherwise, this defeats the function of the API as an abstraction layer.**\n",
```

Comment:
```suggestion
    "**Note: that the tags themselves are specific to each manager, and are not defined by the API or the MediaCreation library. Hosts should not attempt to string munge entity references, and should use the methods covered in this document instead if they wish to obtain references for specific versions. Otherwise, this defeats the function of the API as an abstraction layer.**\n",
```
[consider][minor] Otherwise the comma is ambiguous until you get to "instead" as to whether it's continuing the "should not" or not, which I think is out of the range of pre-scanning and would interrupt read flow.

Reply:
Heeding your more general advice - how about [must](https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/pull/59/commits/8c499276c1c9901ba9c9b0763bb7ebb5b6579703)?